docker-hub/repos/access/index.md

```diff
@@ -22,22 +23,27 @@ You can also assign more granular collaborator rights ("Read", "Write", or
 "Admin") on Docker Hub by using organizations and teams. For more information
 see the [organizations documentation](../../../docker-hub/orgs.md#create-an-organization).
 
-## Viewing repository tags
+### Permissions reference
 
-Docker Hub's individual repositories view shows you the available tags and the
-size of the associated image. Go to the **Repositories** view and select repository to see its tags. To view individual tags, select the **Tags** tab.
+Permissions are cumulative. For example, if you have Read & Write permissions,
+you automatically have Read-only permissions:
 
-![View repo tags](../../images/repo-overview.png)
+- `Read-only` access lets users view, search, and pull a private repository in the same way as they can a public repository.
+- `Read & Write` access lets users pull, push, and view a repository. In addition, it lets users view, cancel, retry or trigger builds
+- `Admin` access lets users pull, push, view, edit, and delete a
+  repository. You can also edit build settings, and update the repositories description, collaborators rights, public/private visibility, and delete.
 
-Image sizes are the cumulative space taken up by the image and all its parent
-images. This is also the disk space used by the contents of the `.tar` file
-created when you `docker save` an image. An image is stale if there has been no push/pull activity for more than one month. For example:
+> **Note**
+>
+> A user who hasn't verified their email address only has
+> `Read-only` access to the repository, regardless of the rights their team
+> membership has given them.
 
-* It hasn't been pulled for more than one month
-* And it hasn't been pushed for more than one month
+## Viewing repository tags
```

Comment:
Saw you updated the conjugation of the other titles. Not sure if this one was missed or intentional.
```suggestion
## View repository tags
```